.github/workflows/workflow-run-docker-rust-build.yaml

```diff
@@ -45,6 +45,11 @@ on:
         required: false
         type: boolean
         description: Whether to build additional testing images. If not specified, only the base release images will be built
+      TARGET_REGISTRY:
```

Comment:
Can this be choice or enum?

Can you validate

Nit: instead of remote call it all?

Reply:
iirc  `choice` is only available in `workflow_dispatch` and not `workflow_call` or something like that

Reply:
RIP

https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype

Alternatively can we validate its valid in a step? that way you can fail early if the value doesnt make sense?